tutorials/equations.ipynb

```diff
@@ -300,7 +300,7 @@
       "        self.reset_state_from_file()\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    poly_ad.jac.todense()[:5, :5]
</pre><p>Should we remove this line in light of the expanded repr of AD arrays? Or use print(poly_ad) above?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1439/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
Changed the code line above to `print(poly_ad)`.
I left this one, since it is a small matrix, and I think it is instructive to see that it is a diagonal matrix, giving a hint on the inner workings of the AD.